app/allocation/controllers/view.js

```diff
@@ -44,7 +56,12 @@ module.exports = function view(req, res) {
       .map(move => ({
         id: move.id,
         person: move.person,
-        card: moveToCardComponent(move),
+        card: moveToCardComponent(
+          {
+            ...move,
+          },
+          removeMoveLink(move)
```

Comment:
This value could be passed directly into the map to avoid having to mess around with the presenters.

Having looked at the presenters, this value isn't send to the `card` component so there is no need to send this value into the presenter.